Dapper.MoqTests/MockDbParameter.cs

```diff
@@ -16,5 +16,10 @@ internal class MockDbParameter : DbParameter
 
         public override void ResetDbType()
         { }
+
+        public override string ToString()
```

Comment:
This was introduced for debugging, it's not required for the change. It doesn't hurt to keep it however.